public/app/plugins/datasource/loki/query_utils.ts

```diff
@@ -1,4 +1,5 @@
 const selectorRegexp = /(?:^|\s){[^{]*}/g;
+const caseInsensitive = '(?i)';
```

Comment:
Needs a comment that this is a mode modifier and that it's not working in JS.

Reply:
Done